tests/unit/form/Validator.spec.ts

```diff
@@ -0,0 +1,95 @@
+import FormValues from '@/form/FormValues';
+import Validator, { ValidationResult } from '@/form/Validator';
+
+describe( 'validator', () => {
+	it( 'full form', () => {
```

Comment:
This could be a bit more verbose. The standard form for these titles is something like
```suggestion
	it( 'returns no errors with a complete form', () => {
```